WARNING  firebolt.common.settings:settings.py:102  Passing connection credentials directly in Settings is deprecated.
 Use Auth object instead.
 Examples:
  >>> from firebolt.client.auth import UsernamePassword
  >>> ...
  >>> settings = Settings(auth=UsernamePassword(username, password), ...)
 or
  >>> from firebolt.client.auth import Token
  >>> ...
  >>> settings = Settings(auth=Token(access_token), ...)
INFO     tests.integration.conftest:conftest.py:27 STOPPED_ENGINE_URL: integration-testing-1684782934-stopped.firebolt.us-east-1.dev.firebolt.io
INFO     tests.integration.conftest:conftest.py:27 ENGINE_NAME: integration_testing__1684782934
INFO     httpx:_client.py:1019 HTTP Request: GET https://api.dev.firebolt.io/iam/v2/account "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1019 HTTP Request: GET https://api.dev.firebolt.io/compute/v1/providers "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1019 HTTP Request: GET https://api.dev.firebolt.io/core/v1/accounts/15ff9dd6-32a6-417b-b004-6f69d7cc4c8b/databases:getIdByName?database_name=integration_testing__1684782934 "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1019 HTTP Request: GET https://api.dev.firebolt.io/core/v1/accounts/15ff9dd6-32a6-417b-b004-6f69d7cc4c8b/databases/7e985983-86f0-4318-b26e-a95438cd5578 "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1019 HTTP Request: GET https://api.dev.firebolt.io/core/v1/accounts/15ff9dd6-32a6-417b-b004-6f69d7cc4c8b/bindings?page.first=5000&filter.id_database_id_eq=7e985983-86f0-4318-b26e-a95438cd5578 "HTTP/1.1 200 OK"
INFO     httpx:_client.py:1019 HTTP Request: GET https://api.dev.firebolt.io/core/v1/accounts/15ff9dd6-32a6-417b-b004-6f69d7cc4c8b/engines/fb702de0-1740-41c3-acc7-366663bff173 "HTTP/1.1 200 OK"